cli/mod.ts

```diff
@@ -65,7 +66,8 @@ const command = new Command()
   .command("node", node)
   .command("tunnel", tunnel)
   .command("completions", new CompletionsCommand())
-  .command("help", new HelpCommand().global());
+  .command("help", new HelpCommand().global())
+  .command("webfinger", webfinger);
```

Comment:
Could you change the order of these lines so that the `help` command appears the last?

Reply:
Sure! Changed at 76bf32a259c7d124bf9f2c37f35555ed03563db2!